test/integration/test_multisharding_one.py

```diff
@@ -93,3 +150,49 @@ def test_router_role(cluster):
         'static': 'aaaaaaaa-aaaa-4000-b000-000000000001',
         'default': 'aaaaaaaa-aaaa-4000-b000-000000000001'
     }
+
+
+def test_set_vshard_options_positive(cluster):
+    obj = edit_vshard_group(cluster, group = "default",
+                            variables = {
+                                'rebalancer_max_receiving': 42,
+                                'collect_lua_garbage': True,
+                                'sync_timeout': 24,
+                                'collect_bucket_garbage_interval': 42.24,
+                                'rebalancer_disbalance_threshold': 14
+                            }
+                            )
+    assert 'errors' not in obj, obj['errors'][0]['message']
+    assert obj['data']['cluster']['edit_vshard_options'] == True
+
+    assert get_vshard_groups(cluster) == \
+        [
+            {
+                'collect_bucket_garbage_interval': 42.24,
+                'collect_lua_garbage': True,
+                'rebalancer_disbalance_threshold': 14,
+                'rebalancer_max_receiving': 42,
+                'sync_timeout': 24,
+                'name': 'default',
+                'bucket_count': 3000,
+                'bootstrapped': True,
+            }
+        ]
```

Comment:
Extend this test: call `edit_vshard_group(sync_timeout = 25, rebalancer_max_receiving = None)`

Check that sync_timeout is updated, other values should remain the same